src/app/archive/archive_lib/extensional.ml

```diff
@@ -56,6 +56,8 @@ module Internal_command = struct
     ; secondary_sequence_no: int
     ; typ: string
     ; receiver: Public_key.Compressed.Stable.Latest.t
+    ; receiver_account_creation_fee_paid:
```

Comment:
This fee is burned, not paid, correct? It might be better to be more explicit.

Reply:
It is burned, that's true, and also true (I believe) for payments, that deduct the account creation fee from the payment amount.

For now, I think it's better to use terminology consistent with the other account creation fee occurrences. It would take a db migration to update the column names.

Reply:
But maybe you're right, we should use "burned" here.

Reply:
I made #9365 to track this